src/main/resources/public/js/view.js

```diff
@@ -16,10 +16,15 @@ var bots = [            // hard-coded bots for testing
     newBot(1,1,0,"bob"), 
     newBot(3,3,0,"bobette")
     ]; 
-const X_RANGE = 11; // x range for grid, TODO: capitalize const
-const Y_RANGE = 11; // y range for grid
-var x_int = 40; // actual spacing between grid lines
-var y_int = 40;
+//const X_RANGE = 11; // x range for grid, TODO: capitalize const
+//const Y_RANGE = 11; // y range for grid  <---- what does this mean?????????
+var viewWidth =  520;
```

Comment:
oh i see you are using this. capitalize the name since it is a constant and use it instead of 520 throughout the code.